vic/drivers/shared_all/src/compute_derived_state_vars.c

```diff
@@ -136,12 +136,12 @@ compute_derived_state_vars(all_vars_struct *all_vars,
                     if (FIRST_VEG) {
                         FIRST_VEG = false;
                         set_node_parameters(soil_con->Zsum_node,
-                                            soil_con->max_moist_node,
+                                            soil_con->porosity_node,
                                             soil_con->expt_node,
                                             soil_con->bubble_node,
                                             soil_con->alpha, soil_con->beta,
                                             soil_con->gamma, soil_con->depth,
-                                            soil_con->max_moist, soil_con->expt,
+                                            soil_con->porosity, soil_con->expt,
```

Comment:
do we only want to be changing `max_moist_node` here or am I missing something?

Reply:
No, both variables changed name, because they both have units of `mm/mm`, which is porosity. The convention now is that `max_moist` (and `max_moist_node`) has units of `mm`. Please see my summary at the top of the PR.

Reply:
Ah ok, thanks for clarifying